Sources/Jetworking/Client/Executor/RequestExecuter/SyncRequestExecuter/RequestOperation.swift

```diff
@@ -1,6 +1,10 @@
 import Foundation
 
-final class RequestOperation : Operation, CancellableRequest {
+class RequestOperation : Operation, CancellableRequest {
+    var identifier: Int {
+        return task?.identifier ?? 0
```

Comment:
Is it possible to make the task not optional? Or do you have an idea of creating a useful identifier? Maybe a random Int between minInt and maxInt or something else? (Collisions would be less frequently)

Reply:
Using `Int.random(in: Int.min..<Int.max)` to prevent collisions.